contrib/src/main/java/com/datatorrent/contrib/avro/AvroToPojo.java

```diff
@@ -88,6 +88,10 @@
   @VisibleForTesting
   int fieldErrorCount = 0;
 
+  public AvroToPojo() {
```

Comment:
This is not needed I thought the problem was that there was a non default constructor, that is not the case.